_posts/2018-08-30-pair-programming.markdown

```diff
@@ -0,0 +1,184 @@
+---
+layout: epic
+title: "The Hows and Whys of Pair Programming"
+date: 2018-08-30
+author: [yuki, orta]
+categories: [pairing, culture]
+---
+
+Why pair program? As new engineers join Artsy we've been experimenting with different programming cultures - Yuki
+came from Pivotal Labs where they have a strong pair programming culture and introduced it at Artsy - it's been
+about a year and a half and we're all really loving the changes he's introduced.
+
+I asked Yuki if he'd pair program with me on a blog, post on Pair Programming, and this is it. So, we're going to
+dive into what pair programming is, why you should do it, what are good mental models to think about, the techniques
+you can use to make it work, what hardware you might need and how Yuki persuaded so many of us to start doing it
+more often.
+
+<!-- more -->
+
+## Why pair program?
+
+The idea of pair programming came from a very popular book called [Extreme Programming Explained: Embrace
+Change][extreme-explained]. Code review has been a good practice in software development. Many organizations have
```

Comment:
This sentence feels a bit incomplete:

> Code review has been a good practice in software development.

Maybe add something like:

> Code review has been a good practice in software development for decades.

Or something?